helm/trivy-operator/templates/kyverno-policy-exception.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.kyvernoPolicyExceptions.enabled }}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: {{ include "trivy-operator-helpers.name" . }}-exception
+  namespace: {{ .Values.kyvernoPolicyExceptions.namespace | default .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
+spec:
+  exceptions:
+    - policyName: disallow-host-namespaces
+      ruleNames:
+        - host-namespaces
+        - autogen-host-namespaces
+        - autogen-cronjob-host-namespaces
+  match:
+    any:
+      - resources:
+          kinds:
+            - Job
+            - Pod
+            - Cronjob
```

```suggestion
            - CronJob
```

This is case-sensitive.